### ISSUE TITLE
tensorrt batchnorm

### ISSUE THREAD

**Issue by Lemonononon** [Lemonononon/deeplearning#1]
https://docs.nvidia.com/deeplearning/tensorrt/developer-guide/index.html#faq

Q: How do I implement batch normalization in TensorRT?
A: Batch normalization can be implemented using a sequence of `IElementWiseLayer` in TensorRT. More specifically:
```
adjustedScale = scale / sqrt(variance + epsilon) 
batchNorm = (input + bias - (adjustedScale * mean)) * adjustedScale
```
